### ISSUE TITLE
usdt.readthedocs.org does not exist

### ISSUE THREAD

**Issue by xen0l** [nshalman/python-usdt#5]
docs/ directory contains simple documentation, but https://usdt.readthedocs.org does not display it at all. It should be fixed. I'll work on some more documentation.

**Comment by nshalman** [comment]
Yeah, the template that I built the repo from includes the skeleton stuff needed for that but I don't think I ever set up an account to get things published.
I'm certainly happy to publish docs if we get them into a usable state.

**Comment by xen0l** [comment]
OK, any ideas what should be covered there? I was thinking about writing some basic stuff about DTrace and how to use pyhton-usdt for now. Do you think that even if it is a single page, it's worth publishing?
